src/bkl/plugins/native.py

```diff
@@ -221,6 +239,8 @@ class NativeCompiledType(TargetType):
 
     use_pic_by_default = True
 
+    link_allow_undefined = False
```

Comment:
Why this? Do you think it's sensible to have different default by different target type in some situations? 

At first glance, I'd say it's better to keep the default same, i.e. False, and so have constant default for the property, and not risk breaking its documentation ("By default, all symbols must be defined…")

Reply:
> Why this? Do you think it's sensible to have different default by different target type in some situations?

Oops, yes, this is a leftover: I thought to allow undefined symbols for loadable modules by default initially, but then changed my mind and decided that it would be best to preserve the current behaviour by default. 

I've simplified this now, thanks for noticing it!